web/js/modules/location-search/util.js

```diff
@@ -154,18 +154,20 @@ export function serializeCoordinatesWrapper(coordinates, state) {
   const { map, proj } = state;
   const serializeCoordinates = (coordinate) => {
     const coordinateValues = [coordinate.latitude, coordinate.longitude];
-    if (map.ui.selected) {
-      const coordinatesWithinExtent = areCoordinatesWithinExtent(proj, coordinateValues);
-      if (!coordinatesWithinExtent) {
-        return;
-      }
-    }
+    if (!map.ui.selected) return null;
+    const coordinatesWithinExtent = areCoordinatesWithinExtent(proj, coordinateValues);
+    if (!coordinatesWithinExtent) return null;
     return coordinateValues;
   };
 
-  const serializeCoordinatesArray = (coordinatesArray) => coordinatesArray.map((coordinate) => serializeCoordinates(coordinate));
+  const serializeCoordinatesArray = (coordinatesArray) => coordinatesArray
+    .map((coordinate) => serializeCoordinates(coordinate))
+    .filter((coordinate) => coordinate !== null);
   const coordinatesURL = Array.isArray(coordinates) ? serializeCoordinatesArray(coordinates) : serializeCoordinates(coordinates);
-  return coordinatesURL.join('+');
+  if (coordinatesURL.length > 0) {
+    const result = coordinatesURL.length === 1 ? coordinatesURL : coordinatesURL.join('+');
```

Comment:
A `.join` call on a single element array just returns that element so I think you could just:

`return coordinatesURL.join('+')`